pingcap/dm
# debug-tools: Binlog event blackhole

### What problem does this PR solve? 

add a _binlog-event-blackhole_ tool to test the speed of fetching binlog event.

### What is changed and how it works?

add a _binlog-event-blackhole_ tool.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
	- run _binlog-event-blackhole_ tool with different args

for more information, see _README.md_ under the _binlog-event-blackhole_ folder.
